Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -401,21 +401,7 @@ public void Run()
             {
                 foreach(var request in _orderRequestQueue.GetConsumingEnumerable(_cancellationTokenSource.Token))
                 {
-                    OrderResponse response;
-                    switch (request.OrderRequestType)
-                    {
-                        case OrderRequestType.Submit:
-                            response = HandleSubmitOrderRequest((SubmitOrderRequest) request);
-                            break;
-                        case OrderRequestType.Update:
-                            response = HandleUpdateOrderRequest((UpdateOrderRequest) request);
-                            break;
-                        case OrderRequestType.Cancel:
-                            response = HandleCancelOrderRequest((CancelOrderRequest) request);
-                            break;
-                        default:
-                            throw new ArgumentOutOfRangeException();
-                    }
+                    var response = HandleOrderRequest(request);
 
                     // we've finally finished processing the request, mark as processed
                     request.SetResponse(response, OrderRequestStatus.Processed);
```

Comment:
Yes, old line 421, new line 407. Without this line, the `HandleOrderRequest` method is incomplete.